sphinx_automodapi/tests/cases/non_ascii/output/api/sphinx_automodapi.tests.example_module.nonascii.NonAsciiÄöüß.rst

```diff
@@ -1,17 +1,17 @@
-NonAscii
-========
+NonAsciiÄöüß
+============
```

Comment:
While it shouldn't be an issue with utf8, I would add more more odd-ball character that manages to mess up systems that can handle latin-1 but not latin-2
```suggestion
NonAsciiÄöüßő
=============
```